packages/flutter_tools/lib/src/cache.dart

```diff
@@ -202,13 +203,17 @@ class FlutterEngine {
     return dirs;
   }
 
-  List<String> _getToolsDirs() {
+  List<List<String>> _getToolsDirs() {
```

Comment:
Could you add a brief doc to this? It looks like it returns a list of tuples of target dir paths and source google storage paths?